### PR TITLE
template for sprint minutes

### PR DESCRIPTION
**Pull request by AClifton22**

### CODE REVIEW

File: doc/sprint1.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Sprint1 Ceremony Minutes
+  
+Date: 2/16/2021
+
+Members present:
+
+* John Cook
```
Comment (johnpcooke94) [inline doc/sprint1.md]: Misspelled my name - should be John Cooke

File: doc/sprint1.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* Zion Emond
+* Alex Stiner
+  
+  ## Demo
+  
+  This sprint, we completed:
+  
+  * Setting up directories and files to be worked in.
+  * Creating and filling out the proposal.md document 
+  * Filling out the READ.md with introduction, prerequisites , initial setup and unit testing
```
Comment (johnpcooke94) [inline doc/sprint1.md]: Typo - README.MD

**Review by johnpcooke94 — CHANGES_REQUESTED** [review]
Just a couple typos to fix

File: doc/sprint1.md (hunk 2 above)
Comment (AClifton22) [reply doc/sprint1.md]: Fixed typos

**Review by johnpcooke94 — APPROVED** [review]

**Pull request merged**